docs/dev-env-setup.md

```diff
@@ -229,12 +314,13 @@ First you should have a look at the logs located in `webcompat.com/tmp`.
 When you start the local server, it will print the location of the logs to the console:
 
 ```bash
-> python run.py
 Statuses Initialization…
-Writing logs to: /Users/acooluser/projects/webcompat.com/tmp
+Writing logs to: /Users/username/projects/webcompat.com/tmp
```

Comment:
nit: not consistent with `/Users/karl/code/webcompat.com/tmp`, but it doesn't really matter. Good first bug?